### DISCUSSION TITLE
How do I convert a document to a PDF file using `pdfcpu`?

### DISCUSSION THREAD

**Discussion by kris3713** [pdfcpu/pdfcpu#668] [Q&A] [answered]
The question is in the title, how can I convert a document file to a PDF file using `pdfcpu`?

**Comment by hhrutter** [comment, marked as answer]
Sorry, no conversion from filetype xyz to PDF supported.
A PDF processor processes PDF input files and that's also the focus of pdfcpu.
In addition you may also use pdfcpu to create straight PDFs or PDF forms using JSON.
